### PR TITLE
[#2627] fix XtendClasspathContainer to work with the new maven based project setup

### PR DESCRIPTION
**Pull request by cdietrich**
[#2627] fix XtendClasspathContainer to work with the new maven based project setup

### CODE REVIEW

File: org.eclipse.xtend.ide/src/org/eclipse/xtend/ide/buildpath/XtendClasspathContainer.java [hunk 1]
```diff
@@ -87,7 +87,16 @@ private IPath bundlePath(Bundle bundle) {
...
+		URL binFolderURL = FileLocator.find(bundle, new Path("target/classes"), null); //$NON-NLS-1$
+		if (binFolderURL != null) {
+			try {
+				URL binFolderFileURL = FileLocator.toFileURL(binFolderURL);
+				return new Path(binFolderFileURL.getPath()).makeAbsolute();
+			} catch (IOException e) {
+				LOG.error("Can't resolve path '" + bundle.getSymbolicName() + "'", e); //$NON-NLS-1$ //$NON-NLS-2$
+			}
+		}
+		binFolderURL = FileLocator.find(bundle, new Path("bin/main"), null); //$NON-NLS-1$
```
Comment (cdietrich) [inline org.eclipse.xtend.ide/src/org/eclipse/xtend/ide/buildpath/XtendClasspathContainer.java:99]: should we remove the two bin options?

Comment (LorenzoBettini) [reply org.eclipse.xtend.ide/src/org/eclipse/xtend/ide/buildpath/XtendClasspathContainer.java:99]: bin/main is history so I'd keep only the new one

**Review by LorenzoBettini — APPROVED** [review]
I seem to understand that this is only for runtime Eclipse, so if you tested it and it works then we can merge this. I'd only remove the old bin main folder as I commented inline.

**Comment by LorenzoBettini** [comment]
Argh... I hadn't noted there's also an attempt with "bin"! That one shouldn't be there either, because .classpath should already be set to generate to target/classes, not to bin.

**Comment by cdietrich** [comment]
ok, i have also removed that one ;)

**Pull request merged**